packaging/package.csproj

```diff
@@ -0,0 +1,48 @@
+<Project Sdk="Microsoft.Build.NoTargets/3.5.6">
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
+        <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+        <IsPackable>true</IsPackable>
+        <NoWarn>NU5100;NU5128</NoWarn>
+
+        <!-- MSBuild tasks shouldn't be referenced. This is by design. -->
+        <NoWarn>NU5100;NU5128</NoWarn>
+        <Authors>Microsoft</Authors>
+        <Company>Microsoft</Company>
+        <Packageid>Microsoft.NET.Build.Containers</Packageid>
+        <Description>Tasks and targets to natively publish .NET applications as containers.</Description>
+        <Copyright>&#169; Microsoft Corporation. All rights reserved.</Copyright>
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+        <PackageProjectUrl>https://github.com/dotnet/sdk-container-builds</PackageProjectUrl>
+        <RepositoryUrl>https://github.com/dotnet/sdk-container-builds</RepositoryUrl>
+        <RepositoryType>git</RepositoryType>
+        <PackageTags>containers;docker;Microsoft.NET.Build.Containers</PackageTags>
+        <PackageReadmeFile>README.md</PackageReadmeFile>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <ProjectReference Include="../containerize/containerize.csproj" ReferenceOutputAssembly="true" />
+        <ProjectReference Include="../Microsoft.NET.Build.Containers/Microsoft.NET.Build.Containers.csproj" OutputItemType="ContainerLibOutput"/>
+        <PackageReference Include="System.CommandLine" IncludeAssets="runtime"/>
+    </ItemGroup>
+
+
+    <Target Name="DoThePackaging" AfterTargets="Build">
+        <ItemGroup>
+            <Content Include="$(OutDir)containerize.dll" Pack="true" PackagePath="containerize\" />
+            <Content Include="$(OutDir)containerize.exe" Pack="true" PackagePath="containerize\" />
```

Comment:
Excellent point, should we set `UseAppHost` to false for the containerize project entirely to prevent an apphost from being part of the build outputs? That might make glob-including the built assets for that project easier.

Reply:
I didn't consider this, good idea! So our tool task should just run `dotnet.exe` (with no path) and the first arg would be `path/to/containerize.dll --plus-all-the-args`, any reason (off the top of your head) that we need to worry about ToolTasks not playing well with calling tools on PATH?